src/vit-servicing-station/vit-servicing-station-lib/src/db/migrations.rs

```diff
@@ -1,20 +1,46 @@
+use tempfile::TempDir;
+
 use crate::db::DbConnection;
 
+const PG_INITIAL_SETUP_UP: &str =
+    include_str!("../../migrations/postgres/00000000000000_diesel_initial_setup/up.sql");
+const PG_SETUP_UP: &str =
+    include_str!("../../migrations/postgres/2022-11-03-210841_setup_db/up.sql");
+const SQLITE_SETUP_UP: &str =
+    include_str!("../../migrations/sqlite/2020-05-22-112032_setup_db/up.sql");
+
 pub fn initialize_db_with_migration(db_conn: &DbConnection) {
     // Discard all output
     let mut sink = std::io::sink();
 
     match db_conn {
-        DbConnection::Sqlite(db_conn) => diesel_migrations::run_pending_migrations_in_directory(
-            db_conn,
-            &std::path::PathBuf::from("./migrations/sqlite"),
-            &mut sink,
-        ),
-        DbConnection::Postgres(db_conn) => diesel_migrations::run_pending_migrations_in_directory(
-            db_conn,
-            &std::path::PathBuf::from("./migrations/postgres"),
-            &mut sink,
-        ),
+        DbConnection::Sqlite(db_conn) => {
+            let t = TempDir::new().unwrap();
+
+            write_tmp_migration(&t, "2020-05-22-112032_setup_db", SQLITE_SETUP_UP);
+
+            diesel_migrations::run_pending_migrations_in_directory(db_conn, t.path(), &mut sink)
+        }
+        DbConnection::Postgres(db_conn) => {
+            let t = TempDir::new().unwrap();
```

Comment:
Maybe we should handle this? instead of unwrap?

Reply:
Agreed

Reply:
I'm not sure we need to use `Result` here.

Presumably this function is being called only at server startup? If so, panicking is arguably the correct behaviour. 

Don't think it matters much, but I like to reserve `Result` for "recoverable errors", and this sort of thing isn't really recoverable

Reply:
agreed, but at least some more meaningful error message instead of unwrap?

Reply:
This is used by tests and by `vit-servicing-station-cli db init`. I like the idea of making it return result and leaving the responsability of panicking to the caller because it isn't necessarily unrecoverable in every context.

I had already started working on making these changes and made a new commit so you can see how it changes.

Reply:
+1

If we need further information, it should probably go to the logs/tracing.